components/automate-ui/src/app/services/chef-session/chef-session.service.ts

```diff
@@ -248,12 +271,12 @@ export class ChefSessionService implements CanActivate {
 
   setDefaultSession(): void {
     localStorage.setItem(sessionKey,
-                         JSON.stringify({
-                           'uuid': 'test_subject',
-                           'username': 'testchefuser',
-                           'fullname': 'Test User',
-                           'groups': ['group1', 'group2', 'group3'],
-                           'id_token': 'test_id_token'
-                         }));
+      JSON.stringify({
+        'uuid': 'test_subject',
```

Comment:
[nit] 
```suggestion
        uuid: 'test_subject',
```

but it really doesn't matter and was like this before.